src/eventshub/src/component/Search/Search.tsx

```diff
@@ -35,21 +35,21 @@ const Search = () => {
   const indexName = algoliaIndexMap[activeTab];
   const activeField = facetFieldNamesMap[activeTab];
   const changeTab = (tab: AppTabs) => dispatch(setActiveTab(tab));
-  const placeholder = (activeTab !== 'vendors') ? `Search by ${activeTab}` : 'Search by Services & Vendors';
+  const placeholder = (activeTab !== 'vendors') ? `Search by ${activeTab.charAt(0).toUpperCase() + activeTab.slice(1)}` : 'Search by Services & Vendors';
 
   return (
-    <div>
-      <Tabs activeTab={activeTab} onClick={changeTab} />
-      <div>
-        <InstantSearch<ExtendedUIState, RouteState>
-          searchClient={algoliaClient}
-          future={{
-            preserveSharedStateOnUnmount: true,
-            persistHierarchicalRootCount: true,
-          }}
-          insights
-        >
-          <Index indexName={indexName} indexId={indexName}>
+    <InstantSearch<ExtendedUIState, RouteState>
+      searchClient={algoliaClient}
+      future={{
+        preserveSharedStateOnUnmount: false,
+        persistHierarchicalRootCount: true,
+      }}
+      insights
+    >
+      <Index indexName={indexName} indexId={indexName}>
+        <div>
+          <Tabs activeTab={activeTab} onClick={changeTab} />
+          <div>
```

Comment:
This is pretty much all just moving the instantsearch and index components up around everything so that the Tabs component has the ability to get to the instantsearch state.